Dockerfile

```diff
@@ -0,0 +1,123 @@
+ARG BASE_IMAGE=nvidia/cuda:11.3.1-devel-ubuntu18.04
+ARG PYTHON_VERSION=3.8
+
+FROM ${BASE_IMAGE} as dev-base
+
+# Install system dependencies
+RUN --mount=type=cache,id=apt-dev,target=/var/cache/apt \
```

Comment:
I wonder if there a image that we could use here instead of building it ourselves?